loopFibonacci.c

```diff
@@ -0,0 +1,16 @@
+/*Sean Kee*/
+/*Loop Fibonacci Sequence*/
+#include <stdio.h>
+
+int main()
+{
+	int fib[20] = {1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89, 144, 233, 377, 610, 987, 1597, 2584, 4181, 6765};
```

Comment:
you didn't do these correctly! you shouldn't have hard coded this - you should have figured out the numbers on your own.